channels/tsmf/client/tsmf_codec.c

```diff
@@ -255,6 +255,119 @@ static const TSMFMediaTypeMap tsmf_sub_type_map[] =
 
 };
 
+
+#define WaveFormatTag(_arg_) _arg_
```

Comment:
I don't see any advantage to this, can you elaborate on why you're using that define and use it for enum values ?